genie-agent/src/test/groovy/com/netflix/genie/agent/execution/process/impl/JobProcessManagerImplSpec.groovy

```diff
@@ -547,12 +548,19 @@ class JobProcessManagerImplSpec extends Specification {
         threadPoolScheduler.shutdown()
     }
 
+
     def "Force kill diehard process"() {
         def future = Mock(ScheduledFuture)
 
+        Path touchedFile = temporaryFolder.resolve("touchz")
+
+        String script = new ClassPathResource("slowlyDie.test.sh")
```

Comment:
Not sure if you're aware but Groovy has nice utilities for reading files... I think there's just a simple `File.text` API that will read the contents into a string